### DISCUSSION TITLE
Resources for cross platform C++ library development

### DISCUSSION THREAD

**Discussion by donnie-jp** [MobileNativeFoundation/discussions#182] [General]
It seems like many mobile products have C++ core cross platform components but it isn't an area that is well covered in typical mobile dev blogs.

Is there anything like iOSDevWeekly / AndroidWeekly but focused on the mobile cpp space?

Can anyone recommend relevant resources? Either company/personal blogs, Slack/Discord, Twitter, anything really!

**Comment by donnie-jp** [comment]
A few that I know of:

* https://mobilecpp.slack.com/
* https://djinni.xlcpp.dev
* https://devblogs.microsoft.com/cppblog/
* https://discourse.cmake.org (specific to cmake it seems)
* https://blog.conan.io (specific to Conan package manager)
* https://blog.jetbrains.com/clion/
* https://github.com/rigtorp/awesome-modern-cpp (general C++)

I'll expand this list as I find more.

**Comment by keith** [comment]
I haven't really seen resources around this, but would love to see more. At Lyft we're also currently experimenting with Rust on mobile in use cases where previously we might have used C++ instead.

**Comment by ahmdyasser** [comment]
If there any resources for iOS specifically this would be great!
